### PR TITLE
added lambda store

### PR DESCRIPTION
**Pull request by smithdave**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -679,6 +679,7 @@ Table of Contents
    * [FaunaDB](https://fauna.com/) — Serverless cloud database, with native GraphQL, multi-model access and daily free tiers up to 5GB
    * [graphenedb.com](https://www.graphenedb.com/) — Neo4j as a service, up to 1,000 nodes and 10,000 relations free
    * [heroku.com](https://www.heroku.com/) — PostgreSQL as a service, up to 10,000 rows and 20 connections free (provided as an "addon," but can be attached to an otherwise empty app and accessed externally)
+   * [lambda.store](https://lambda.store/) — Serverless Redis with free tier up to 5000 requests per day and 256MB max database size
```
Comment (ripienaar) [inline README.md:682]: no TLS in the free tier? That's just irresponsible, so I am afraid I wont be adding this here.

Comment (svenanderson) [reply README.md:682]: @ripienaar  Can you please remove Redislabs from the list then? They do not have TLS in their free plan. Even they do not have TLS in their paid Essentials plan. Thank you.

https://redislabs.com/redis-enterprise/deployments-features/  
Screen Shot 2020-04-20 at 4 27 55 PM

Comment (ripienaar) [reply README.md:682]: will do

Comment (ripienaar) [reply README.md:682]: https://github.com/ripienaar/free-for-dev/pull/1484

**Pull request closed without merging**